<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MovimentacaoDoPaciente</class>
 <widget class="QDialog" name="MovimentacaoDoPaciente">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>458</width>
    <height>258</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Movimentação do Paciente</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Movimentação do Paciente</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_data">
          <property name="text">
           <string>Data</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateEdit" name="dateEdit"/>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_situacao">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Situação</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_situacao">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <item>
           <property name="text">
            <string>Início do programa de hemodiálise - Crônico</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Mudança para diálise peritoneal</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Retorno ao programa de diálise peritoneal</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Retorno ao programa de hemodiálise - Crônico</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Transferência para outra clínica de hemodiálise</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Transferência para o programa de hemodiálise - Agudo</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Transferência para outro hospital para procedimentos</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Transplante renal</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Abandono do tratamento</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Óbito</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Outros</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_Outros"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_confirmar">
          <property name="text">
           <string>Confirmar</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_cancelar">
          <property name="text">
           <string>Cancelar</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
